src/electron/adapters/electron-storage-adapter.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { IndexedDBDataKeys } from '../../background/IndexedDBDataKeys';
+import { StorageAdapter } from '../../common/browser-adapters/storage-adapter';
+import { IndexedDBAPI } from '../../common/indexedDB/indexedDB';
+
+export class ElectronStorageAdapter implements StorageAdapter {
+    constructor(private readonly indexedDBInstance: IndexedDBAPI) {}
+    public setUserData(items: Object, callback?: () => void): void {
+        // tslint:disable-next-line:no-floating-promises - top-level entry points are intentionally floating promises
```

Comment:
isn't worth it to at least log the error instead of disabling the tslint rule?

Reply:
The tslint error is not for the error but that I am not `awaiting` the promise. But, I will verify and may be use `logger` to log this.

Reply:
so I check this:
```// tslint:disable-next-line:no-floating-promises - top-level entry points are intentionally floating promises```
is not needed if you add a `.catch(...)` at the end of the promise. This was my original point... also, the last part of the comment on tslint disable (```top-level entry points are intentionally floating promises```) doesn't apply on this case, basically this is not a top-level entry point

Reply:
Agreed. Updated.